evalml/pipelines/components/transformers/imputers/simple_imputer.py

```diff
@@ -44,24 +66,19 @@ def fit(self, X, y=None):
 
         Returns:
             self
+
         """
         X = infer_feature_types(X)
+
         nan_ratio = X.ww.describe().loc["nan_count"] / X.shape[0]
         self._all_null_cols = nan_ratio[nan_ratio == 1].index.tolist()
 
-        # Not using select because we just need column names, not a new dataframe
-        natural_language_columns = [
-            col
-            for col, ltype in X.ww.logical_types.items()
-            if type(ltype) == NaturalLanguage
-        ]
-        if natural_language_columns:
-            X = X.ww.copy()
-            X.ww.set_types({col: "Categorical" for col in natural_language_columns})
+        X, _ = self._drop_natural_language_columns(X)
```

Comment:
Similar to Freddy's comment, do we use the second output ever or can we remove it altogether?

Reply:
We can, it was the result of my over-engineering and failure to scale all the way back *heavy breathing*

Reply:
<img src="https://media0.giphy.com/media/ncsQI9sF3pa8w/giphy.gif"/>